### PR TITLE
Fix timers not triggering a second time

### PR DESCRIPTION
**Pull request by cartolari**

### CODE REVIEW

**Comment by albertossilva** [comment]
It would be nice if you confirm that the unscribe is null after the process.

**Comment by cartolari** [comment]
I updated the PR with the specs

File: src/IntervalFormatter.test.js [hunk 1]
```diff
@@ -83,9 +83,13 @@ describe('IntervalFormatter', () => {
       .returns(unsubscriber)
```
Comment (mvcds) [inline src/IntervalFormatter.test.js]: I think it worths to test with the [sinon's faker timer](http://sinonjs.org/releases/v1.17.7/fake-timers/) if the time has advanced automatically

Comment (cartolari) [inline src/IntervalFormatter.test.js]: Maybe, but I think we should leave it to another PR since this one is basically to ensure the old timer is cleaned up independently of it being triggered or not
